src/System.DirectoryServices.Protocols/tests/DirectoryAttributeModificationCollectionTests.cs

```diff
@@ -0,0 +1,173 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+
+namespace System.DirectoryServices.Protocols.Tests
+{
+    public class DirectoryAttributeModificationCollectionTests
```

Comment:
We can't, as each method is not actually an ICollection method, it is just defined as it's own method. This is because each collection derives from `CollectionBase`